<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Firefox Profil Baszkuráló</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="searchLayout">
        <item>
         <widget class="QLabel" name="searchLabel">
          <property name="text">
           <string>Search title:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="searchEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="categoryLabel">
          <property name="text">
           <string>Cat.:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="categoryLine"/>
        </item>
        <item>
         <widget class="QPushButton" name="searchButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="placeTable">
        <property name="contextMenuPolicy">
         <enum>Qt::ActionsContextMenu</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionOpenUrl">
   <property name="text">
    <string>Open url</string>
   </property>
  </action>
  <action name="actionCopyLink">
   <property name="text">
    <string>Copy link</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>searchEdit</tabstop>
  <tabstop>categoryLine</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>placeTable</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>searchEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>597</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>1148</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>categoryLine</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>860</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>1148</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
